pkg/api/convertertoapi.go

```diff
@@ -498,6 +498,7 @@ func convertV20170701WindowsProfile(v20170701 *v20170701.WindowsProfile, api *Wi
 func convertVLabsWindowsProfile(vlabs *vlabs.WindowsProfile, api *WindowsProfile) {
 	api.AdminUsername = vlabs.AdminUsername
 	api.AdminPassword = vlabs.AdminPassword
+	api.ImageVersion = vlabs.ImageVersion
```

Comment:
We'll need the same conversion in the reverse direction in `pkg/api/converterfromapi.go`.